src/helio.jl

```diff
@@ -0,0 +1,139 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _helio(jd::T, num::Integer, radian::Bool) where {T<:AbstractFloat}
```

Comment:
Just putting it as 1,2,3... seemed a more simpler way of representing it and to avoid smallcase/uppercase problems and keyerrors problems